lib/charliecloud.py

```diff
@@ -1458,6 +1458,12 @@ def file_gzip(path, args=[]):
       unlink(path_c)
    # Compress.
    cmd([file_gzip.gzip] + args + [str(path)])
+   # Zero out GZIP header timestamp data.
+   zeds = bytes.fromhex('00 00 00 00')
```

Comment:
Note this can be written as `b'\x00\x00\x00\x00'`, if you prefer.

Also, I don't think you need this variable. You can just pass it directly to `write()` below.